ocw_data_parser/ocw_data_parser_test.py

```diff
@@ -491,38 +491,43 @@ def test_course_features(ocw_parser):
     assert ocw_parser.parsed_json["course_features"] == [
         {
             "ocw_feature": "AV special element video",
-            "ocw_subfeature": "Tutorial",
+            "ocw_feature_notes": "",
             "ocw_feature_url": "./resolveuid/b5785e071ddb991cf3dfd7cc469e6397",
             "ocw_speciality": "",
-            "ocw_feature_notes": "",
+            "ocw_subfeature": "Tutorial",
+            "course_feature_tag": "Tutorial Videos",
         },
         {
             "ocw_feature": "AV lectures",
-            "ocw_subfeature": "Video",
+            "ocw_feature_notes": "",
             "ocw_feature_url": "./resolveuid/6b1f662457366951bfe85945521b0299",
             "ocw_speciality": "",
-            "ocw_feature_notes": "",
+            "ocw_subfeature": "Video",
+            "course_feature_tag": "Lecture Videos",
         },
         {
             "ocw_feature": "Assignments",
-            "ocw_subfeature": "problem sets with solutions",
+            "ocw_feature_notes": "",
             "ocw_feature_url": "./resolveuid/87609dbba9d13a6b234d62de21a20433",
             "ocw_speciality": "",
-            "ocw_feature_notes": "",
+            "ocw_subfeature": "problem sets with solutions",
+            "course_feature_tag": "Problem Sets with Solutions",
         },
         {
             "ocw_feature": "Exams",
-            "ocw_subfeature": "Solutions",
+            "ocw_feature_notes": "",
             "ocw_feature_url": "./resolveuid/c13c4766c0cf1486f0cf6435c531eaad",
             "ocw_speciality": "",
-            "ocw_feature_notes": "",
+            "ocw_subfeature": "Solutions",
+            "course_feature_tag": "Exams with Solution",
         },
         {
             "ocw_feature": "Instructor Insights",
-            "ocw_subfeature": "",
+            "ocw_feature_notes": "",
             "ocw_feature_url": "./resolveuid/3f3b7835cf477d3ba10b05fbe03cbffa",
             "ocw_speciality": "",
-            "ocw_feature_notes": "",
+            "ocw_subfeature": "",
+            "course_feature_tag": "???",
```

Comment:
I was going to open a separate issue for handling `"ocw_feature": "Instructor Insights"` -- is that data mixed in with the other course feature data?

Reply:
in what way "instructor Insights" is different ?

Reply:
I believe there are two types of "Instructor Insights." One is a course feature with a `"ocw_feature_url"` that points to the course's Instructor Insights page. This PR is handling that fine, as far as I can tell. 

There's a separate vocabulary in the raw JSON `"tcATmit_instructional_tags"` Instructional tags, used to create and populate the Instructor Insights filter on https://ocw.mit.edu/educator/ . I'm still trying to understand the need for these, so they are out of scope for this PR.